### PR TITLE
GIBCT: Accordion Header Font is Incorrect on the Profile Page #19923

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
On the institution profile page, each section is divided into collapsable accordions. The headers of these sections should be in Bitter typeface but have changed to Sans Serif. This is present on all comparison tool profile pages including VET TEC.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19923

## Testing done
Local Testing

## Screenshots
Screen Shot 2019-11-07 at 10 53 22 AM

## Acceptance criteria
- [x] Font is updated to Bitter with serif typeface
- [x] Accordion button is 59px high

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
   .accordion-button-text {
     font-size: 1.35em;
+    font-family: 'bitter', serif;
```
Comment (bkjohnson) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: Is there a reason that applying the `.vads-u-font-family--serif`class wouldn't work?  More info [here](https://design.va.gov/utilities/font-family).

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -43,6 +44,8 @@
 
   button.usa-accordion-button {
     text-align: left;
+    margin-top: 0.208em;
+    margin-bottom: 0.208em;
```
Comment (bkjohnson) [inline src/applications/gi/sass/partials/_gi-profile-page.scss]: Same kind of thing here - what about using the `.vads-u-margin-y--0p25` class on the button?  Here are some more [options](https://design.va.gov/utilities/margins#margin-top-and-bottom)

Comment (ksantiagomorgan6) [reply src/applications/gi/sass/partials/_gi-profile-page.scss]: vads-u-margin-y--0p25 is too small and vads-u-margin-y--0p5 is too big. Had to make custom margins on this one.

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/sass/partials/_gi-profile-page.scss]: Applied these changes

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**